tests/unit/TestCase.php

```
@@ -15,58 +16,104 @@ class TestCase extends \PHPUnit\Framework\TestCase
      */
-    protected function mockClient(array $requestData, array $responseData): MockObject
-    {
-        return $this->mockEntity(RabbitMQClient::class, [
-           [
-               'methodName' => 'request',
-               'inputData'  => $requestData,
-               'outputData' => $responseData
-           ]
-        ]);
+    protected function mockClient(
+        array $requestData,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$requestData'. (undefined)

(UnusedFormalParameter)